### DISCUSSION TITLE
Right handed traffic for drawing multiple roads

### DISCUSSION THREAD

**Discussion by OBRND** [liodali/osm_flutter#320] [Q&A]
I would like to ask if a RHT(Right handed traffic) configuration could be implemented on the multiple road function. It is hard to work with a left sided configuration in a RHT country, so is there a tweak that I don't currently know about that does this or is this strictly meant for LHT countries?

**Comment by liodali** [comment]
can you provide more information

**Reply by OBRND** [reply]
The function below draws roads/routes on the set marker but it draws it according to a left handed traffic configuration meaning they draw routes on the left lane of roads. which is the opposite of how traffic flows in my country. It was designed for countries like the UK and I didn't find any parameters that could be set to change the current configuration. below is the portion of the code that performs what I explained above. any help is much appreciated.

 final configs = [
            MultiRoadConfiguration(
              startPoint: GeoPoint(
                latitude: (p.latitude),
                longitude: p.longitude,
              ),
              destinationPoint: GeoPoint(
                latitude: services.fu,
                longitude: services.gu,
              ),
            ),
            MultiRoadConfiguration(
                startPoint: GeoPoint(
                  latitude: p.latitude+0.0005,
                  longitude: (p.longitude+0.0005),
                ),
                destinationPoint: GeoPoint(
                  latitude: services.fu,
                  longitude: services.gu,
                ),
                roadOptionConfiguration: MultiRoadOption(
                  roadColor: Colors.orange,
                )),
            MultiRoadConfiguration(
              startPoint: GeoPoint(
                latitude: p.latitude+0.001,
                longitude: p.longitude+0.001,
              ),
              destinationPoint: GeoPoint(
                latitude: services.fu,
                longitude: services.gu,
              ),
              roadOptionConfiguration: MultiRoadOption(
                roadColor: Colors.blue,
              ),)
          ];
          final listRoadInfo = await controller.drawMultipleRoad(
            configs,
            commonRoadOption: MultiRoadOption(
              roadColor: Colors.red,
              roadType: RoadType.car,
            ),
          );

**Reply by liodali** [reply]
did you try to make roadwidth bigger 
because in lib I use osrm api that support by default righthand traffic

**Reply by OBRND** [reply]
No luck there. when I set the roadWidth it didn't even draw the road on map. there is some problem there. It didn't work on any value. i/m not even sure if it's my error or the API's problem.
